src/Mvc/test/Microsoft.AspNetCore.Mvc.Core.Test/Infrastructure/ConfigureCompatibilityOptionsTest.cs

```diff
@@ -31,10 +31,12 @@ public void PostConfigure_NoValueForProperty_DoesNothing()
         public void PostConfigure_ValueIsSet_DoesNothing()
         {
             // Arrange
-            var configure = Create(CompatibilityVersion.Version_2_0, new Dictionary<string, object>()
-            {
-                { nameof(TestOptions.TestProperty), true },
-            });
+            var configure = Create(
+                CompatibilityVersion.Version_3_0,
+                new Dictionary<string, object>
+                {
+                    { nameof(TestOptions.TestProperty), true },
+                });
```

Comment:
I don't understand why we need this level of cosmetic churn

Reply:
I don't know why the indentation got cleaned up here. But "this level" has only two instances I can see in this PR.